src/app/controllers/components/ControllerHeaderForms/AddController/AddController.tsx

```diff
@@ -31,7 +35,7 @@ export const AddController = ({ clearHeaderContent }: Props): JSX.Element => {
       <CodeSnippet
```

Comment:
Do we not have any state in the UI that lets it know deb vs. snap? How are end users supposed to know which of these they have?

Reply:
I don't think we do unfortunately. I think the best that we can do for now is to display a dropdown, like we do in our main docs.